SC-10203: Initial docs for private locations (#305)
sematext/docs: docs/synthetics/private-locations.md

title: Private Locations

> description: Private locations allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc

Reviewer: Private locations allows => Private locations functionality allows
etc => etc.

> description: Private locations functionality allows you to monitor the APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc.

Reviewer 2: ```suggestion
description: The private locations feature allows you to monitor APIs and webpages that are not accessible from the Internet such as your internal websites, private APIs, etc.
```

> Private locations allow you to run the synthetics tests from your private network. Some of the use cases for using private locations are:

Reviewer: synthetics tests => synthetic tests

> Private locations functionality allows you to run the synthetic tests from your private network. Some of the use cases for using private locations are:

Reviewer 2: ```suggestion
The private locations feature allows you to run synthetic tests from your private network. Some of the use cases for using private locations are:
```

* Monitor internal APIs or web applications that are not accessible from the Internet
* Compare the performance of your applications between the Internet & your private network
* Monitor your applications from specific locations that are not supported by Sematext

## Create Private Location

To create a private location, navigate to Synthetics -> Private Locations -> New Private Location. 

> ![Create Private Location]()

Reviewer 2: No link?

* **Name** - Unique name for the private location
* **Description** - A brief description of the location

> * **City** - Select the city where the machine running the agent is physically located. This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city.

Reviewer: This is used to calculate latitude/longitude to plot the location in the map. If your city is not listed, select the nearest city. => If your city is not listed, select the nearest city. This is used to calculate latitude/longitude to plot the location in the map. 
(just swapping sentences, a little easier for reader to follow)

Each private location is assigned a unique token. You can disable or delete a location, when not in use.

## Install Private Agents

> After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images one for each HTTP & Browser monitor respectively. 

Reviewer: There are two Docker images one for each HTTP & Browser monitor respectively.
=>
There are two Docker images, one for HTTP monitor and another for Browser monitor.

> After creating the private location from UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images, one for HTTP monitor and another for Browser monitor.

Reviewer 2: ```suggestion
After creating the private location from the UI, you need to install the agents on a machine in your private network. Private agents are deployed as Docker containers. There are two Docker images, one for HTTP monitors and another for Browser monitors.
```

> You can deploy both agents or one based on your monitor types. Follow the instructions in the UI to install private agents in your network.

Reviewer: > You can deploy both agents or one based on your monitor types

Not sure what you mean by "on your monitor types"?

Maybe this should be:
You can deploy either the HTTP or Browser monitor or both.
?

Author: Yes. I will change to "You can deploy either the HTTP or Browser monitor or both."

> ![Agent Installation Instructions]()

Reviewer 2: No link?

> If you have multiple private locations, you need to install agents individually in each location. Private agents use the HTTPS protocol to periodically fetch the monitors to run and send the result to Sematext Cloud. If you have a firewall configured in your network, you need to open a bidirectional HTTPS traffic for `synthetics-scheduler.sematext.com` and `synthetics-receiver.sematext.com` hosts. For the EU region, the hosts are `synthetics-scheduler.eu.sematext.com` and `synthetics-receiver.eu.sematext.com`.

Reviewer 2: ```suggestion
If you have multiple private locations, you need to install agents individually in each location. Private agents use the HTTPS protocol to periodically fetch the monitors to run and send the result to Sematext Cloud. If you have a firewall configured in your network, you need to open bidirectional HTTPS traffic for the `synthetics-scheduler.sematext.com` and `synthetics-receiver.sematext.com` hosts. For the EU region, the hosts are `synthetics-scheduler.eu.sematext.com` and `synthetics-receiver.eu.sematext.com`.
```

## Using Private Locations

> Once you create a private location from UI and install the agents, you can use it while creating the monitors or update existing monitors to run from the private location. Private location behaves exactly like public locations. 

Reviewer 2: ```suggestion
Once you create a private location from the UI and install the agents, you can use it while creating the monitors or update existing monitors to run from the private location. Private locations behave exactly the same as public locations. 
```

![Create Monitor with Private Location](../images/synthetics/private-locations-monitor.png)

> ![Monitor Overview with Private Location]()

Reviewer 2: No link?
